ui/app/templates/components/transit-key-action/decrypt.hbs

```diff
@@ -33,6 +36,27 @@
         </div>
       </div>
     {{/if}}
+    {{#if (or (eq @key.type "rsa-2048") (eq @key.type "rsa-3072") (eq @key.type "rsa-4096"))}}
```

Comment:
I find this easier to read and update in the future, but no need to change - mostly an FYI
```suggestion
    {{#if (includes @key.type (array "rsa-2048" "rsa-3072" "rsa-4096"))}}
```